production-dependency-templates/rds/config.yml.mustache

```diff
@@ -0,0 +1,5 @@
+service-environment-variables:
+  - RDS_HOST: {{db-name}}.{{app-name}}.local
+  - {{service-env-var-db-name}}: {{db-name}}
+  - {{service-env-var-username}}: {{username}}
+  - {{service-env-var-password}}: {{secret(password-secret-name)}}
```

Comment:
These fields will be rendered with a combination of regular Mustache string interpolation and method interpolation